app/assets/scripts/components/widgets/DistrictField.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import districtNames from '../../utils/districtNames';
+
+  /**
+   * District widget with Governorate & district dropdowns
+   */
+export default class DistrictField extends React.Component {
+  constructor (props) {
+    super(props);
+    if (props.formData && props.formData.governorate && props.formData.district) {
+      const {governorate, district} = props.formData;
+      this.state = {governorate, district};
+    } else {
+      this.state = {governorate: '-1', district: '-1'};
```

Comment:
@kamicut is there a reason you're using a string instead of a falsey value like `null`?